packages/devtools-launchpad/src/utils/defer.js

```diff
@@ -1,12 +1,29 @@
-module.exports = function defer() {
-  let resolve, reject;
-  let promise = new Promise(function() {
-    resolve = arguments[0];
-    reject = arguments[1];
+/* @flow */
+
+declare var Resolve: (result: any) => void;
+declare var Reject: (result: any) => void;
+
+export type Defer = {
+  resolve: Resolve,
+  reject: Reject,
+  promise: Promise<any>
+};
+
+function defer(): Defer {
+  let resolve: Resolve; // eslint-disable-line no-unused-vars
```

Comment:
Is this `eslint-disable` really needed?  It seems to me that `resolve` and `reject` are actually used.